### PR TITLE
feat: adding all option and fixing some styling

### PR DESCRIPTION
**Pull request by anandtiwary**
## Description
Please include a summary of the change, motivation and context.

feat: adding "all" option and fixing some styling

### Testing
Please describe the tests that you ran to verify your changes. Please summarize what did you test and what needs to be tested e.g. deployed and tested helm chart locally. 

### Checklist:
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

### Documentation
Make sure that you have documented corresponding changes in this repository or [hypertrace docs repo](https://github.com/hypertrace/hypertrace-docs-website) if required.

### CODE REVIEW

File: projects/components/src/multi-select/multi-select.component.scss [hunk 1]
```diff
@@ -54,7 +54,6 @@
 .multi-select-content {
   @include dropdown();
   min-width: 120px;
-  max-height: 204px;
```
Comment (aaron-steinfeld) [inline projects/components/src/multi-select/multi-select.component.scss:57]: Shouldn't it have some max height? I'd rather it scroll then run off the screen.

File: projects/components/src/multi-select/multi-select.component.ts [hunk 2]
```diff
@@ -40,6 +40,13 @@ import { SelectSize } from '../select/select-size';
         </ht-popover-trigger>
         <ht-popover-content>
           <div class="multi-select-content">
+            <div class="multi-select-option all-options" (click)="this.onAllSelectionChange()">
```
Comment (aaron-steinfeld) [inline projects/components/src/multi-select/multi-select.component.ts:43]: Can we make the presence of the all option controlled by an input?

Comment (anandtiwary) [reply projects/components/src/multi-select/multi-select.component.ts:43]: sure

File: projects/components/src/multi-select/multi-select.component.scss (hunk 1 above)
Comment (anandtiwary) [reply projects/components/src/multi-select/multi-select.component.scss:57]: How do we choose this value? Different use cases may show different set of options. Do you want to expose an input for this?

Comment (anandtiwary) [reply projects/components/src/multi-select/multi-select.component.scss:57]: Or may be we can set a max height which is equal to available height, that is, from trigger element's position to the bottom on the window.

Comment (anandtiwary) [reply projects/components/src/multi-select/multi-select.component.scss:57]: Let me create a separate story for this change.

**Review by aaron-steinfeld — APPROVED** [review]

**Pull request merged**